test/parallel/test-readline-interface.js

```diff
@@ -175,6 +175,59 @@ function isWarned(emitter) {
   assert.equal(callCount, expectedLines.length);
   rli.close();
 
+  // \t when there is no completer function should behave like an ordinary 
+  //   character
+  fi = new FakeInput();
+  rli = new readline.Interface({ input: fi, output: fi, terminal: true });
+  expectedLines = ['\t'];
```

Comment:
Using an array with `expectedLines` is probably unnecessary, you could just `assert.equal(line, '\t')` in the callback and use a truthy/falsy `called`.